README.md

```diff
@@ -1,2 +1,27 @@
 # schema-test-data
 This repository is for the test [meta]data associated with the latest updates of the HCA metadata schema.
+
+# Brief SOP on generating test data 
+
+1. Download the excel spreadsheet located in this folder (schema_test_data.xlsx) and modify it to include the new metadata-schema changes. 
```

Comment:
```suggestion
1. Download the excel spreadsheet located in this folder ([TestData.xlsx](TestData.xlsx)) and modify it to include the new metadata-schema changes. 
```

Reply:
TestData.xlsx is the outdated version of the test data that does not include recent changes.